packages/fxa-payments-server/src/store/selectors.ts

```diff
@@ -0,0 +1,41 @@
+import { Selector, Plan } from './types';
+
+export const profile: Selector = state => state.api.profile;
+
+export const token: Selector = state => state.api.token;
+
+export const subscriptions: Selector = state => state.api.subscriptions;
+
+export const plans: Selector = state => state.api.plans;
+
+export const customer: Selector = state => state.api.customer;
+
+export const createSubscriptionStatus: Selector = state =>
+  state.api.createSubscription;
+
+export const cancelSubscriptionStatus: Selector = state =>
+  state.api.cancelSubscription;
+
+export const reactivateSubscriptionStatus: Selector = state =>
+  state.api.reactivateSubscription;
+
+export const updatePaymentStatus: Selector = state => state.api.updatePayment;
+
+export const plansByProductId: Selector = state => (
+  productId: string
+): Array<Plan> => {
+  const fetchedPlans = plans(state).result || [];
+  return fetchedPlans.filter((plan: Plan) => plan.product_id === productId);
```

Comment:
This was:
```
    return productId
      ? fetchedPlans.filter((plan: Plan) => plan.product_id === productId)
      : fetchedPlans;
```
... but I took out the check for `productId` because `productId` is required (line 41 in `Product/index.tsx`) so we shouldn't need the ternary.